flytekit/annotated/interface.py

```diff
@@ -258,10 +274,14 @@ def t(a: int, b: str) -> Dict[str, int]: ...
     if hasattr(return_annotation, "__origin__") and return_annotation.__origin__ is tuple:
         # Handle option 3
         logger.debug(f"Task returns unnamed typing.Tuple {return_annotation}")
+        if len(return_annotation.__args__) == 1:
```

Comment:
I thought this is the only way people can name outputs... or are we saying they can NOT name outputs if they return a single output?

I guess I'm confused because I know we allow named tuples with a single element... so I'm not sure what this is...

Reply:
no these are not typing.NamedTuples, which we do accept.  These are typing.Tuples, which we don't.